apps/src/turtle/turtle.js

```diff
@@ -1616,7 +1616,7 @@ Artist.prototype.step = function (command, values, options) {
  */
 function scaleToBoundingBox(maxSize, width, height) {
   if (width < maxSize && height < maxSize) {
-    return {width: width, height: height};
+    return {width: maxSize, height: maxSize};
```

Comment:
This will work for square images, but what about images that don't have aspect ratio of 1?

Let's add some unit tests for this method to help characterize it.